### ISSUE TITLE
Troubleshooting Evernote Imports

### ISSUE THREAD

**Issue by CdtDelta** [standardnotes/forum#283]
Hello,
I'm using the Evernote tool to convert my data over to Standard Notes, and I have one Notebook with a lot (1100+) of notes in it.  I've been breaking down the export in Evernote into smaller groups to take care of the file size, but I ran into one export group where the tools is throwing an error and has a pop up to contact support.
I'm guessing there's something in one of the notes I'm trying to convert that the Standard Notes tool doesn't like.  I wanted to know if there's a log or some other way of isolating the problem note(s) without doing them one at a time (like a way to convert with logging or something like that).

Thanks....

**Comment by mobitar** [comment]
The best chance of a log we have is to open the browser developer tools console, and see what error JavaScript is throwing. If you can paste that output here, I can try to determine what's causing the issue and hopefully get a fix out.

**Comment by CdtDelta** [comment]
Ok I'll see if I can get to that this week.

**Comment by CdtDelta** [comment]
Ok it looks like the problem was I had a couple of notes in Evernote in a row that were basically PDF files (large ones).  When I broke that part down and did it one by one I was able to get them all to export.

It doesn't appear that the size of the export was the problem, I think it was more the fact it had to process multiple Evernote documents with PDF's embedded in them in a row, some being rather large (100 MB).

But it's going through now if you want to close the ticket (unless there's something else you want me to do).
